en/docs/graphql-api.md

```diff
@@ -0,0 +1,79 @@
+# GraphQL APIs
+
+Explore how you can easily design, develop, and manage GraphQL APIs in Choreo to implement your business API strategy.
+
+!!! tip
+    GraphQL API is currently available in Choreo as a preview feature. For instructions to enable it, see [Preview Features](references/preview-features.md).
+
+## What is a GraphQL API?
+
+[GraphQL](https://graphql.org) is a query language for APIs and a runtime for fulfilling those queries with your existing data. A GraphQL API is a category of APIs that supports this query language.
```

Comment:
Can we do a bit of rewording here to avoid using the same introduction as the official site? WDYT?

Reply:
How about the following?

```suggestion
[GraphQL](https://graphql.org) is a query language for APIs to fetch and persist data. A GraphQL API is a category of APIs that supports this query language.
```

Reply:
Done